cmd/crank/beta/render/runtime_docker.go

```diff
@@ -225,7 +225,12 @@ func (r *RuntimeDocker) Start(ctx context.Context) (RuntimeContext, error) {
 	if r.Stop {
 		stop = func(ctx context.Context) error {
 			err := c.ContainerStop(ctx, rsp.ID, container.StopOptions{})
-			return errors.Wrap(err, "cannot stop Docker container")
+			if err != nil {
+				return errors.Wrap(err, "cannot stop Docker container")
+			}
+
+			err = c.ContainerRemove(ctx, rsp.ID, container.RemoveOptions{Force: true, RemoveVolumes: true})
+			return errors.Wrap(err, "cannot remove Docker container")
```

Comment:
Should this be optional? I could imagine wanting to inspect the container after the fact for debugging.

Perhaps remove it by default, with an opt-out flag?

Reply:
Cool, I can update this. Good point raised.

Reply:
Added in a Remove option, which is default. People can override this by 

```
apiVersion: pkg.crossplane.io/v1beta1
kind: Function
metadata:
  name: function-go-templating
  annotations.render.crossplane.io/runtime-docker-cleanup: "Stop"
spec:
  package: xpkg.upbound.io/crossplane-contrib/function-go-templating:v0.4.1
```